Dockerfile

```diff
@@ -44,15 +30,26 @@ RUN apk update && \
     && echo "gem: --no-ri --no-rdoc" > /etc/gemrc \
     && gem update --system \
     && gem install bundler \
-    && bundle install --system $BUNDLER_ARGS \
-    && apk del .ruby-builddeps \
-    && rm -rf /var/cache/apk/*
+    && bundle install --system $BUNDLER_ARGS
```

Comment:
can you please add the `--clean` and `--no-cache ` parameters to bundle install? This should clean stuff up afterwards (not really related to this change but some nice addons)